### PR TITLE
Best. Feature. Eva.

### PR DESCRIPTION
**Pull request by ruprict**
Team, this is my great feature. I made for great success.

### CODE REVIEW

File: glenns_feature.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<html>
+  <head> </head>
+  <body>
+    <h1>OMGEEEE SO ORSOME!</h1>
```
Comment (emmagoodrich) [inline glenns_feature.html:4]: You can't spell

Comment (AJayll) [inline glenns_feature.html:4]: 1 too many Es on 'OMGEEEE'

Comment (easysketchfail) [reply glenns_feature.html:4]: yay

Comment (ruprict) [reply glenns_feature.html]: WTF, how's that mr. perfect?

**Review by easysketchfail — COMMENTED** [review]
cool

**Review by emmagoodrich — APPROVED** [review]

**Review by AJayll — APPROVED** [review]

**Pull request merged**